src/examples/zkapps/hello-world/run.ts

```diff
@@ -27,7 +27,7 @@ txn = await Mina.transaction(feePayer1.publicKey, () => {
   AccountUpdate.fundNewAccount(feePayer1.publicKey);
   zkAppInstance.deploy();
 });
-await txn.sign([feePayer1.privateKey, zkAppPrivateKey]).send();
+await txn.sign([feePayer1.privateKey, zkAppPrivateKey]).sendOrThrowIfError();
```

Comment:
@MartinMinkov are there any other examples in the the examples folder that should be updated with the changes to the simulated blockchain similar to this?